Files/Views/MainPage.xaml.cs

```diff
@@ -69,6 +69,20 @@ public MainPage()
             }
 
             AllowDrop = true;
+
+            AppInstances.CollectionChanged += AppInstances_CollectionChanged;
+        }
+
+        private void AppInstances_CollectionChanged(object sender, System.Collections.Specialized.NotifyCollectionChangedEventArgs e)
+        {
+            if (e.OldItems != null)
```

Comment:
What do you think about possibility to move this to tab close event?

Reply:
I think the `TabStrip_TabCloseRequested` event is not called when the tab is closed by Ctrl+W
But perhaps I could move the logic to `horizontalTabView_TabItemsChanged/verticalTabView_TabItemsChanged` which are triggered by the `TabItemsChanged` event?

Reply:
I think it is possible to find a better solution, but now we can keep this solution.